uisdk/src/test/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenPaymentPresenterTest.kt

```diff
@@ -25,10 +32,10 @@ import org.mockito.junit.MockitoJUnitRunner
 @RunWith(MockitoJUnitRunner::class)
 class AdyenPaymentPresenterTest {
 
+    private var context: Context = mock()
     private var paymentsService: PaymentsService = mock()
     private var userStore: UserStore = mock()
-    private var savedPaymentInfo: SavedPaymentInfo = mock()
-    private var paymentView: BookingPaymentMVP.View = mock()
+    private var paymentView: PaymentDropInMVP.Actions = mock()
```

Comment:
would it be better to call it paymentViewActions?

Reply:
Will resolve in the next PR with the additional unit tests